blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -227,6 +327,26 @@ exports.fetchLogsInjectStart = function fetchLogsInjectStart(url, start, onSucce
             }
         });
 };
+
+/**
+ * Determines if the provided object is a "run" type
+ */
+function isRun(obj) {
+    return obj._class === 'io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl'
```

Comment:
Is there a capability for the run type? Are the capabilities documented anywhere, btw? @imeredith